gc/verbose/schema.xsd

```diff
@@ -452,6 +454,11 @@ xmlns:vgc="http://www.ibm.com/j9/verbosegc">
 		<attribute name="classloader" type="integer" use="required" />
 	</complexType>
 
+	<complexType name="continuation-objects">
+		<attribute name="total" type="integer" use="required" />
+		<attribute name="started" type="integer" use="required" />
```

Comment:
started probably means only active (thus excluding those not only started by also ended). 
not sure if active is the right term though (as per specs)
@babsingh @fengxue-IS, are you ok with 'started' or have some other preference?

Reply:
Yes. In the context of Continuations, `STARTED` looks good; it corresponds the Continuation state where a stack is always populated.